packages/twenty-server/src/engine/api/graphql/graphql-query-runner/resolvers/graphql-query-find-many-resolver.service.ts

```diff
@@ -139,12 +144,30 @@ export class GraphqlQueryFindManyResolverService
       args.filter ?? ({} as Filter),
     );
 
+    const selectedAggregatedFields = this.getSelectedAggregatedFields({
+      objectFields: Object.values(
+        Object.fromEntries(
+          Object.entries(objectMetadataMapItem.fields).filter(
+            ([key, _value]) => !isUuid(key), // remove objectMetadataMapItem fields duplicates
```

Comment:
This feels a bit ugly, did you consider solving the root cause in `generateObjectMetadataMap` directly, introducing 2 maps instead of 1? I know it might be a largr refactor with many downstream consequences but it's probably  better than this uuid hack which we'll have to pay back one day? 
Note: We'll need to make sure to clear all the metadata cache when we deploy if we go with the "clean" way

Reply:
When I say 2 objects it's 2 different map for object level but also have 2 properties instead of one (for the field map by id + field map by name) within the ObjectMetadataMapItem

Reply:
I am not sure how this map with ids as key are used today, I know its for perf improvement reasons. @charlesBochet has more context and probably knows better if a refactor is desirable i think ! 
I think you understood that from the review but to give context, I introduced the filter here not to duplicate the aggregated fields computed (otherwise we would find "minCreatedAt" twice for instance), which we could also do differently not to allow a duplicated value in the selectedAggregatedFields array, which I had done initially and is maybe less ugly (but still a bit hacky)!

Reply:
Yes I understand! Thank you. But  we should introduce two maps on byName and one byId. It doesn't make sense to mix both and lose information, to then re-create it later